src/components/Navbar/__snapshots__/index.test.js.snap

```diff
@@ -220,7 +219,6 @@ exports[`Navbar Component matches the tree 1`] = `
         >
           <svg
             aria-hidden="true"
-            className=""
```

Comment:
why is there a change in navbar here

Reply:
when I run test, it throws some errors, so I clicked "a" to update all tests. But why it changes in other components I have no idea. I had a meeting with Ammar this morning and he told me it should be ok.  do you have any recommendation for this ?